draft-link-v6ops-464xlaton.xml

```diff
@@ -146,28 +146,51 @@ The metrics of IPv4 routes SHOULD be consistent with the metrics of IPv6 default
+In particular, queries for AAAA resource records of "ipv4only.arpa." MUST be sent to the recursive resolvers provided by the network, not to the resolvers configured manually, local recursive resolvers etc.
+</t>
+<t>
+The node SHOULD enable CLAT after discovering the NAT64 prefix, unless by that time the node has already obtained an IPv4 address.
+The node SHOULD NOT wait for an explicit (DHCP Option 108, <xref target="RFC8925"/>) or an implicit (DHCP timeouts) indication that native IPv4 connectivity is not available.
+Such delay would be impactful for IPv4-only applications, as such applications cannot benefit from 464XLAT until CLAT is operational.
+Therefore, the node SHOULD enable CLAT as soon as network support for PLAT is detected while IPv4 connectivity is not yet detected. If IPv4 connectivity becomes available later, the node SHOULD disable CLAT as discussed in the following section.
 </t>
 <t>
 The node MUST follow recommendations from Section 4 of <xref target="RFC7335"/> to avoid IPv4 address space conflicts.
 </t>
-<section anchor="ipv4detect">
-<name>Detecting IPv4 Presence</name>
+
+</section>
+
+<section anchor="disable">
+<name>Disabling CLAT</name>
 <t>
 From a performance perspective, native IPv4 connectivity is preferrable over 464XLAT, so CLAT SHOULD NOT be enabled if the node has IPv4 connectivity over the given interface.
-However, SLAAC might complete faster than DHCPv4. 
-The node SHOULD NOT wait for an explicit (DHCPv4 Option 108, <xref target="RFC8925"/>) or an implicit (DHCPv4 timeouts) indication that native IPv4 connectivity is not available.
-Such delay would be impactful for IPv4-only applications, as such applications cannot benefit from 464XLAT until CLAT is operational.
-Therefore, the node SHOULD enable CLAT as soon as network support for PLAT is detected while IPv4 connectivity is not yet detected. Clients SHOULD then disable CLAT if native IPv4 connectivity is established whether or not PLAT support from the network remains available.
 </t>
-</section>
+
+<t>
+It is possible that after the CLAT instance has started, native IPv4 becomes available (e.g. an IPv4 address received via DHCP).
+The node SHOULD disable CLAT immidiately upon obtaining an IPv4 address via DHCP or manual configuration.
+</t>
+<t>
+While disabling CLAT is impactful for all applications and traffic flows already utilizing CLAT, it is recommended not only from performance perspective, but also from security point of view.
+IPv4-only networks might also lack IPv6 layer2 security features, such as RA Guard. 
+An attacker can send an RA containing the PREF64 option, while the network doesn't provide any PLAT functionality.
+If the node keeps CLAT enabled and uses it for IPv4-only destinations, that traffic will be blackholed.
+As rogue RAs can also disturb traffic to IPv6-enabled destinations, it is very much desirable to ensure that IPv4 traffic is unaffected.
```

Comment:
```suggestion
Rogue RAs can also disturb traffic to destinations that support both IPv4 and IPv6 by causing IPv6 through an attacker's PLAT to be used instead of the legitimate network owner's IPv4 path.
```

Is this the takeaway?